docs/styleguide/maintenance-notes.md

```diff
@@ -9,27 +9,21 @@
    - SEMVER/minor
    - SEMVER/major
 
-## Before tagging a release releasing: bump the package version 
-1. open a PR to bump the package version: e.g. `git checkout -b v0.8.12`
-2. bump the package version number that matches the `SEMVER/patch | SEMVER/minor | SEMVER/major`
-   - e.g. `SEMVER/patch`: `0.8.11` would become -> `0.8.12`
```

Comment:
This might also be helpful in the new version. Which tag leads to which version number.